projects/igniteui-angular/src/lib/grids/filtering/excel-style/grid.excel-style-filtering.component.ts

```diff
@@ -318,9 +318,11 @@ export class IgxGridExcelStyleFilteringComponent implements OnDestroy, AfterView
      * @hidden @internal
      */
     ngAfterViewInit(): void {
-        requestAnimationFrame(() => {
-            this.excelStyleSearch.searchInput.nativeElement.focus();
-        });
+        if (this.excelStyleSearch.column) {
+            requestAnimationFrame(() => {
+                this.excelStyleSearch.searchInput.nativeElement.focus();
```

Comment:
You can move this code to the initialize method since it is called only for the internal filtering.

Reply:
Done